docker/actinia-core/Dockerfile

```diff
@@ -64,12 +64,9 @@ RUN grass --tmp-location EPSG:4326 --exec g.version -rge
 WORKDIR /src
 # Install the d.rast.multi module to render several maps at once
 RUN git clone https://huhabla@bitbucket.org/huhabla/d_rast_multi.git d.rast.multi
-RUN grass --tmp-location EPSG:4326 --exec g.extension -s extension=d.rast.multi url=/src/d.rast.multi
 
-# install some GRASS GIS addons
-RUN grass --tmp-location EPSG:4326 --exec g.extension -s extension=i.cutlines && \
-    grass --tmp-location EPSG:4326 --exec g.extension -s extension=r.colors.out_sld && \
-    grass --tmp-location EPSG:4326 --exec g.extension -s extension=v.centerpoint
+COPY docker/actinia-core/grass_addons_list.csv
```

Comment:
Should we also provide a `<DST>` e.g.

`COPY docker/actinia-core/grass_addons_list.csv /src`

or 

`COPY docker/actinia-core/grass_addons_list.csv .`

?

Reply:
fixed